test/tests/functional/pbs_power_provisioning_cray.py

```diff
@@ -861,19 +800,19 @@ def test_power_ramp_down_nodes(self):
         """
         self.setup_power_ramp_rate()
         a = {"power_ramp_rate_enable": True, 'node_idle_limit': '30'}
-        self.modify_hook_config(attrs=a, id='PBS_power')
+        self.modify_hook_config(attrs=a, hook_id='PBS_power')
         a = {'freq': 60}
         self.server.manager(MGR_CMD_SET, PBS_HOOK, a, id='PBS_power',
                             sudo=True)
         a = {'enabled': 'True'}
         self.server.manager(MGR_CMD_SET, PBS_HOOK, a, id='PBS_power',
                             sudo=True)
-        t = int(time.time())
-        self.logger.info("Waiting for 12 mins to ramp down all the nodes")
-        time.sleep(720)
-        for n in self.server.status(NODE):
-            if 'resources_available.PBScraynid' in n:
-                self.server.expect(NODE, {'state': 'sleep'}, id=n['id'])
+        self.logger.info("Waiting for 15 mins to ramp down all the nodes")
+        time.sleep(900)
```

Comment:
I don't see you extending the timeout for this test.  Won't it timeout after 3 minutes?

Reply:
I have added timeout to all test expect node_sort_key test.